test.js

```diff
@@ -0,0 +1,16 @@
+function main(a, b) {
+  switch (cond) {
+  case "one":
+    doSomething(); // JSHint will warn about missing 'break' here.
+  case "three":
+    doSomethingElse();
+  case "two":
```

Comment:
We'll work on this, I agree that's more clear then commenting on the commit itself.